pkg/agent/agentserver/tunnel.go

```diff
@@ -121,7 +121,10 @@ func (t *Tunnel) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				},
 			},
 		}
-		t.Server.Backend.Send(packet)
+		err = t.Server.Backend.Send(packet)
+		if err != nil {
+			klog.Errorf("Error sending packet %v", err)
```

Comment:
Might want to continue here. It would prevent any confusion from logging about the packet having been (successfully) forwarded.

Reply:
:+1: